xdcr/rosmar_xdcr.go

```diff
@@ -135,33 +135,31 @@ func (r *rosmarManager) processEvent(ctx context.Context, event sgbucket.FeedEve
 				base.TracefCtx(ctx, base.KeyWalrus, "Skipping replicating doc %s, _vv.cas %d <= %d", docID, event.Cas, toCas)
 				return true
 			}
-			fmt.Printf("sourceHLV: %+v\nsourceMou: %+v\ntoCas: %d\n", sourceHLV, sourceMou, toCas)
 			if base.HexCasToUint64(sourceMou.CAS) <= toCas {
 				r.targetNewerDocs.Add(1)
 				base.TracefCtx(ctx, base.KeyWalrus, "Skipping replicating doc %s, _mou.cas %d <= %d", docID, event.Cas, toCas)
 				return true
 			}
 		}
-		if targetXattrs == nil {
-			targetXattrs = make(map[string][]byte, 1) //  length for _vv
+		var newXattrs map[string][]byte
+		if sourceXattrs == nil {
+			newXattrs = make(map[string][]byte)
+		} else {
+			newXattrs = sourceXattrs
+			for _, xattrName := range []string{base.VvXattrName, base.MouXattrName, base.SyncXattrName} {
+				delete(newXattrs, xattrName)
```

Comment:
It looks like we're already deleting _vv and _mou in getBodyHLVAndMou.  Do you think it would make the implementation more straightforward clear if the sourceXattrs returned by getBodyHLVAndMou were called 'non-mobile xattrs', and excluded _vv, _mou and _sync?